@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp

```diff
@@ -800,6 +800,20 @@ class CfgVehicles
 				shortcut="Binocular";
 				statement="[this] call weaponRocketsalvo";
 			};
+			class hellfireTraj
+			{
+				displayName="";
+				useAction=false;
+				showSwitchAction=false;
+				position="zamerny";
+				onlyForPlayer=1;
+				radius=8;
+				showWindow=0;
+				priority=0;
+				condition="(player == driver this || player == gunner this) && ([this, 0] call fza_fnc_mpdGetCurrentDisplay == ""wpn"") && (currentweapon this isKindOf [""fza_hellfire"", configFile >> ""CfgWeapons""])";
```

Comment:
You might want to not check if the weapon page is open, might be disorientating when you're heads in the ORT and can't see the MPDs